api/v1alpha1/addon_types.go

```diff
@@ -134,40 +142,158 @@ func (ClusterAutoscaler) VariableSchema() clusterv1.VariableSchema {
 	}
 }
 
+type DefaultStorage struct {
+	ProviderName           string `json:"providerName"`
+	StorageClassConfigName string `json:"storageClassConfigName"`
+}
+
 type CSIProviders struct {
```

Comment:
Can we please change the name of this to CSI to be consistent with CNI, CPI, etc?

Reply:
So the difference between those and this is that we are able to have multiple CSIProviders for a cluster. 

I think the name `CSI` just implies one?

Reply:
Not in my head, it says to me this is `Container Storage Interface` configuration, which then embeds multiple providers. But I'm not gonna for this too hard :)